api/pkg/transformer/feast/bigtablestore/encoding_test.go

```diff
@@ -0,0 +1,458 @@
+package bigtablestore
+
+import (
+	"context"
+	"reflect"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/bigtable"
+
+	feast "github.com/feast-dev/feast/sdk/go"
+	"github.com/feast-dev/feast/sdk/go/protos/feast/serving"
+	"github.com/feast-dev/feast/sdk/go/protos/feast/types"
+	"github.com/gojek/merlin/pkg/transformer/spec"
+	"github.com/golang/protobuf/proto"
+	"github.com/linkedin/goavro/v2"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+type InMemoryRegistry struct {
```

Comment:
Why do we create `InMemoryRegistry` for test? It currently make the `CachedCodecRegistry` untested.